dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2461,10 +2461,14 @@ def test_getitem_optimization(tmpdir, engine, preserve_index, index):
     dsk = optimize_dataframe_getitem(out.dask, keys=[out.key])
 
     read = [key for key in dsk.layers if key.startswith("read-parquet")][0]
-    subgraph = dsk.layers[read]
-    assert isinstance(subgraph, DataFrameIOLayer)
-    assert subgraph.columns == ["B"]
-    assert next(iter(subgraph.dsk.values()))[0].columns == ["B"]
+    subgraph_rd = dsk.layers[read]
```

Comment:
```suggestion
    subgraph_rd = hlg_layer(dsk, "read-parquet")
```
https://github.com/dask/dask/blob/a504115312b1e591604276a3098b2e0076bf49e1/dask/utils_test.py#L134-L139

Reply:
cool!